src/experiments/metis_test.rs

```diff
@@ -74,7 +74,7 @@ mod tests {
     use metis::option::Seed;
     use metis::{Graph, Idx};
 
-    use crate::experiments::metis_test::{partition, Link, Node};
```

Comment:
Ahh, my code formatter in CLion did that - it's the Save Actions plugin. Are you using another one? I can switch to yours.

Reply:
I use rustfmt. It comes with the cargo toolchain. You can enable it in clion in settings -> Langauages & Frameworks -> Rust -> Rustfmt